core/envelope-sim/src/main/java/fr/sncf/osrd/envelope_sim/pipelines/MaxEffortEnvelope.java

```diff
@@ -48,50 +47,51 @@ public static Envelope addAccelerationCurves(
             cursor.findPosition(overlayBuilder.getLastPos());
             builder.addPart(partBuilder.build());
         }
-        while (cursor.findPartTransition(MaxSpeedEnvelope::increase)) {
-            var partBuilder = new EnvelopePartBuilder();
-            partBuilder.setAttr(EnvelopeProfile.ACCELERATING);
-            var overlayBuilder = new ConstrainedEnvelopePartBuilder(
-                    partBuilder,
-                    new SpeedConstraint(0, FLOOR),
-                    new EnvelopeConstraint(maxSpeedProfile, CEILING)
-            );
-            var startSpeed = maxSpeedProfile.interpolateSpeedLeftDir(cursor.getPosition(), 1);
-            var startPosition = cursor.getPosition();
-            EnvelopeAcceleration.accelerate(context, startPosition, startSpeed, overlayBuilder, 1);
-            cursor.findPosition(overlayBuilder.getLastPos());
-            builder.addPart(partBuilder.build());
-        }
-        return builder.build();
-    }
 
-    /** Generate overlays everywhere the train cannot physically maintain the target speed */
-    public static Envelope addMaintainSpeedCurves(
-            EnvelopeSimContext context,
-            Envelope maxSpeedProfile
-    ) {
-        var rollingStock = context.rollingStock;
-        var path = context.path;
-        var builder = OverlayEnvelopeBuilder.forward(maxSpeedProfile);
-        var cursor = EnvelopeCursor.forward(maxSpeedProfile);
-        while (cursor.findPart(MaxEffortEnvelope::maxEffortPlateau)) {
-            double speed = cursor.getStepBeginSpeed();
-            var tractiveEffortCurve = context.tractiveEffortCurveMap.get(cursor.getPosition());
-            assert tractiveEffortCurve != null;
-            double maxTractionForce = PhysicsRollingStock.getMaxEffort(speed, tractiveEffortCurve);
-            double rollingResistance = rollingStock.getRollingResistance(speed);
-            double inertia = rollingStock.getInertia();
-            double worstRamp = Math.asin((maxTractionForce - rollingResistance) / inertia / 9.81) * 1000;
-            var envelopePart = cursor.getPart();
-            while (!cursor.hasReachedEnd() && cursor.getPart() == envelopePart) {
-                double highRampPosition = path.findHighGradePosition(
-                        cursor.getPosition(), envelopePart.getEndPos(), rollingStock.getLength(), worstRamp);
-                cursor.findPosition(highRampPosition);
-                if (cursor.getPosition() == envelopePart.getEndPos())
-                    break;
+        while (!cursor.hasReachedEnd()){
```

Comment:
No space between `)` and `{`? I'm surprised this doesn't cause a checkstyle error

Reply:
Fixed !